bootstrap/pgbouncer/main.tf

```diff
@@ -18,12 +18,30 @@ variable "dbsync_probe_image_tag" {
   default = "27a9dbc30253e7d2036f1d6648d406f3d17a90e2"
 }
 
-variable "pg_bouncer_replicas" {
-  default = 1
+variable "dns_zone" {
+  default = "demeter.run"
+}
+
+variable "extension_name" {
+  default = "dbsync-v3"
+}
+
+variable "instance_role" {
+  default = "pgbouncer"
+}
+
+variable "network" {
+  type    = string
+  default = "preview"
+}
```

Comment:
Given that the network and the dns_zone are only going to be used to generate the dns, I think we should change this to receive the dns directly, instead of hardcoding a way to template it from the parameters. This goes better with the way we are doing it in txpipe with random domain names